packages/suite-desktop/src-electron/libs/processes/TorProcess.ts

```diff
@@ -16,6 +16,7 @@ class TorProcess extends BaseProcess {
     controlPort: number;
     torHost: string;
     authFilePath: string;
+    bootstrap = { progress: '', summary: '' };
```

Comment:
question: is this actually assigned anywhere?

Reply:
yes, it was left over from previous approach, not needed anymore 6431feff2d6de0d22ab85580ce4c170d6d96ae63